### ISSUE TITLE
Frame less ZNC client

### ISSUE THREAD

**Issue by d4708** [zeronet-conservancy/zeronet-conservancy#76]
What about make laconic alternative for the current UI implementation?

I see just many issues in the code, but mostly I'm interested to write something that not requires iframe JS.
Also we can implement new, zero-id and other centralization-less solutions there.

The dreams about new interface will be described in this subject.

**Comment by d4708** [comment]
The reason of initiation this idea - UI code quality, #53, #45 and other issues I not very interested to resolve.

So why not

**Comment by caryoscelus** [comment]
> Frame less

what do you mean by that ? if you mean dropping iframes , they're for security with the whole badly written js api . we're planning to eventually get rid of it via replacing it with safe non-js api that would allow js-less dynamic sites , console interface , js dynamic sites all with the same client code

i've been writing about this general direction on and off in different topics , hard to keep track of all of them

**Comment by d4708** [comment]
> dropping iframes

exactly, I'm noticed about the issues just a thoughts. if that is duplicate feel free to close this subject.
just plan to explain implementation later as learning.

**Comment by d4708** [comment]
Suppose it is a long story, but not so long like #53

**Comment by caryoscelus** [comment]
actually i suppose it's a much longer story . #53 assumes the current interface existing , whilst we intend to rewrite everything at some point . currently i'm experimenting with safe haskell api . draft will be up in `dev` or other branch soon

**Comment by caryoscelus** [comment]
btw , also there's /raw/ prefix if you don't need extra flashy 0net functionality

**Comment by caryoscelus** [comment]
should be possible and nice to implement with znapi development
